tests/test_cache.c

```diff
@@ -1,5 +1,8 @@
 
 void test_cache_invalidate(TestContext *ctx) {
+	disable_interrupts();
+	DEFER(enable_interrupts());
```

Comment:
Maybe leave a comment here on why we need this. Does this fix the failures with everdrive BTW?

Reply:
I don't have a failing test at the moment unfortunately so I cannot verify if it totally fixes them. I'll keep an eye though.